src/clustering.jl

```diff
@@ -96,8 +97,9 @@ Create 'nboth' substations with both load and generation, chosen randomly.
 function cluster_load_gen!(grid::Grid, nboth)
     gens = [bus for bus in buses(grid) if isa(bus, GenBus)]
     nb = 0
-    srand(grid.seed + 66) # Just so we don't keep returning to the same point.
+    Random.seed!(grid.seed + 66) # Just so we don't keep returning to the same point.
```

Comment:
Instead of doing this I would recommend doing:
```julia
using Random: seed!
```
As this lets developers know what functions you are using from `Random`